style.css

```diff
@@ -61,4 +76,17 @@ input[type='number']::-webkit-inner-spin-button {
 
 input[type='number'] {
     -moz-appearance: textfield;
+}
+
+.input-group .input-group-addon {
+    background-color: #D8E5EF;
+}
+
+.section hr {
+    border-bottom: 1px solid #66B9BF;
+    border-top: 1px solid #B2DCDF ;
+}
+
+.damageTable .table-striped>tbody>tr:nth-of-type(odd) {
```

Comment:
we should stay consistent in spacing of the >, either spaces on either side or none anywhere